### ISSUE TITLE
hi, can you share the train/val/test raw images, better on BaiDuYun, you know, i can not download from google drive in China. thanks a lot.

### ISSUE THREAD

**Issue by BecketXu** [hezhangsprinter/DCPDN#7]

**Comment by hezhangsprinter** [comment]
Sorry, baiduyun  account is not easy to have without Chinese Phone number. 
But you can also generate the sample using 'create_train.py' 
(Please download the NYU-depth @ http://horatio.cs.nyu.edu/mit/silberman/nyu_depth_v2/nyu_depth_v2_labeled.mat)

**Comment by BecketXu** [comment]
yes, thanks for quick reply!

**Comment by 3togo** [comment]
I can't download from below:
http://horatio.cs.nyu.edu/mit/silberman/nyu_depth_v2/nyu_depth_v2_labeled.mat
any other alternative link available?

**Comment by zhaoxin111** [comment]
> will it be too big for baiduyun?下午5:22, 2018年9月18日, Matrix <notifications@github.com>: I can't download from below: http://horatio.cs.nyu.edu/mit/silberman/nyu_depth_v2/nyu_depth_v2_labeled.mat any other alternative link available? 用迅雷 —You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
> […](#)
> -- Sent from Yandex.Mail for mobile

2.8G

**Comment by noobgrow** [comment]
> are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute t

你好，能不能把训练集上传百度云啊，下不下来，总会断。迅雷也不行。

**Comment by shawnyuen** [comment]
> 请问有没有下载成功的，能不能把训练集上传百度云啊，下不下来，总会断。迅雷也不行。

download 'nyu_depth_v2_labeled.mat' via wget. The training set is so large (～80GB), generating training and validation set via 'create_train.py' is fast.

**Comment by MrLuoJia** [comment]
Does anybody run the code successfully? I am trying to run it, but I encountere a lot of errors. Can anyboy help me?
有人弄这个弄成功了的么？可以分享一下经验么。
